contrib/building.md

```diff
@@ -167,6 +166,9 @@ git clone -c core.symlinks=true https://github.com/realm/realm-js
 
 or manually create the symlinks using directory junctions if you already have the repo cloned:
 
+******
+*TODO: Needs updating:*
```

Comment:
Consider putting these in markdown comments? Or do you plan to resolve these before merging?

Reply:
I'll go ahead and update those to MD comments, it was mainly to make it hard to miss for reviewers. I'll leave it in for an upcoming iteration that we can look at when people are back from the holidays 👍 🎅